src/kernel-install/90-loaderentry.install.in

```diff
@@ -39,13 +39,28 @@ else
     ENTRY_DIR="${ENTRY_DIR_ABS#"$BOOT_MNT"}"
 fi
 
+KERNEL_DEST="$ENTRY_DIR_ABS/linux"
+KERNEL_ENTRY="$ENTRY_DIR/linux"
+LOADER_ENTRY="$BOOT_ROOT/loader/entries/$ENTRY_TOKEN-$KERNEL_VERSION.conf"
+
+plugin_init() { :;}
+plugin_remove_entry() { :;}
+plugin_update_options() { :;}
+plugin_rename_initrd() { :;}
+plugin_finish() { :;}
+
+if [ -e /usr/lib/kernel/loaderentry-plugin.sh ]; then
+    # shellcheck source=/dev/null
+    . /usr/lib/kernel/loaderentry-plugin.sh
+fi
```

Comment:
Should this maybe check `$KERNEL_INSTALL_CONF_ROOT` and also be overridable with `/etc/kernel/loaderentry-plugin.sh`?

Also some bikeshedding: I'm not sure "plugin" is a good name, because the kernel-install man page refers to the files like this one as plugins (see e.g. the `$KERNEL_INSTALL_PLUGINS` environment variable)

Reply:
suggestions welcome :-) Not sure overriding makes too much sense but sure, could be done.

Reply:
I guess this could be similar to how `install.conf` is looked up (see `kernel-install.in`
```shell
if [ -n "$KERNEL_INSTALL_CONF_ROOT" ]; then
    install_conf="$KERNEL_INSTALL_CONF_ROOT/install.conf"
elif [ -f "/etc/kernel/install.conf" ]; then
    install_conf="/etc/kernel/install.conf"
elif [ -f "/usr/lib/kernel/install.conf" ]; then
    install_conf="/usr/lib/kernel/install.conf"
else
    install_conf=
fi
```